apple/internal/partials/resources.bzl

```diff
@@ -142,6 +142,33 @@ def _validate_processed_locales(*, label, locales_dropped, locales_included, loc
                   sets.str(unused_locales) + " in locale filter. Please verify " +
                   "apple.locales_to_include is defined properly.")
 
+        if locales_requested and locales_to_exclude:
+            conflicting_locales = sets.intersection(locales_requested, locales_to_exclude)
+            if sets.length(conflicting_locales):
+                # There is no way to issue a warning, so print is the only way
+                # to message.
+                # buildifier: disable=print
+                print("Warning: " + str(label) + " dropping " +
```

Comment:
@keith, do we have a precedent for this in the codebase?

Reply:
seems like we should just fail?